aea/test_tools/config.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the config of the AEA test framework."""
+
+from aea.cli_gui import DEFAULT_AUTHOR
```

Comment:
@panasevychol I removed this from your previous PR as it creates an unnecessary duplication of the config imports. check out what's on develop. The only constants not covered by my amendment are `DEFAULT_PRIVATE...` which you should import from wherever it is declared in the configs, same for FETCHAI.